### ISSUE TITLE
ocrmypdf sends non rotated (_ocr.png) file to tesseract if image contains EXIF Rotation Information

### ISSUE THREAD

**Issue by qupfer** [jbarlow83/OCRmyPDF#668]
Hi, 
maybe I'm using it wrong, maybe a bug.

**Describe the bug**
The input image and the output pdf is in correct orientation, but the 000001_ocr.png is not,
with the result, that the text layer is crap.
It looks like its because the raw image is not stored faced up, but include a correct rotation exif information. 

**To Reproduce**

```bash
ocrmypdf -v 1-k -l deu --image-dpi 72 --rotate-pages input.jpg out.pdf 
```
stderr:
```bash
ocrmypdf 11.3.1
Running: ['tesseract', '--list-langs']
Running: ['tesseract', '--version']
Found tesseract 4.1.1
Running: ['tesseract', '-l', 'deu', '--print-parameters', 'pdf']
Running: ['gs', '--version']
Found gs 9.53.3
pikepdf mmap disabled
os.symlink(Downloads/Wein Schinken Waffeln.jpg, /tmp/com.github.ocrmypdf.lrokrlx6/origin)
Input file is not a PDF, checking if it is an image...
Input file is an image
Input image has no ICC profile, assuming sRGB
Image seems valid. Try converting to PDF...
imgformat = JPEG
input dpi = 72 x 72
rotation = 90°
input colorspace = RGB
width x height = 4032px x 3024px
read_images() embeds a JPEG
Successfully converted to PDF, processing...
pikepdf mmap disabled
Using Tesseract OpenMP thread limit 3
pikepdf mmap disabled
    1 Running: ['gs', '-dQUIET', '-dSAFER', '-dBATCH', '-dNOPAUSE', '-sDEVICE=jpeggray', '-dFirstPage=1', '-dLastPage=1', '-r150.000000x150.000000', '-o', '-', '-sstdout=%stderr', '-dAutoRotatePages=/None', '-f', '/tmp/com.github.ocrmypdf.lrokrlx6/origin.pdf']
    1 Rotating output by 0
    1 Running: ['tesseract', '-l', 'osd', '--psm', '0', '/tmp/com.github.ocrmypdf.lrokrlx6/000001_rasterize_preview.jpg', 'stdout']
    1 with existing rotation ⇨, page is facing ⇧, confidence 18.10 - rotation appears correct
    1 Rasterize with png16m, rotation 0
    1 Running: ['gs', '-dQUIET', '-dSAFER', '-dBATCH', '-dNOPAUSE', '-sDEVICE=png16m', '-dFirstPage=1', '-dLastPage=1', '-r150.000000x150.000000', '-o', '-', '-sstdout=%stderr', '-dAutoRotatePages=/None', '-f', '/tmp/com.github.ocrmypdf.lrokrlx6/origin.pdf']
    1 Rotating output by 0
    1 resolution (150, 150)
    1 Rotating 90
    1 Running: ['tesseract', '-l', 'deu', '-c', 'textonly_pdf=1', PosixPath('/tmp/com.github.ocrmypdf.lrokrlx6/000001_ocr.png'), '/tmp/com.github.ocrmypdf.lrokrlx6/000001_ocr_tess', 'pdf', 'txt']
    1 Text rotation: (text, autorotate, content) -> text misalignment = (0, 0, 90) -> 270
    1 Grafting
    1 Page rotation: (content, auto) -> page = (90, 0) -> 90
Postprocessing...
os.symlink(/tmp/com.github.ocrmypdf.lrokrlx6/graft_layers.pdf, /tmp/com.github.ocrmypdf.lrokrlx6/fix_docinfo.pdf)
Running: ['gs', '-dQUIET', '-dBATCH', '-dNOPAUSE', '-dSAFER', '-dCompatibilityLevel=1.6', '-sDEVICE=pdfwrite', '-dAutoRotatePages=/None', '-sColorConversionStrategy=RGB', '-dAutoFilterColorImages=true', '-dAutoFilterGrayImages=true', '-dJPEGQ=95', '-dPDFA=2', '-dPDFACompatibilityPolicy=1', '-o', '-', '-sstdout=%stderr', '/tmp/com.github.ocrmypdf.lrokrlx6/fix_docinfo.pdf', '/tmp/com.github.ocrmypdf.lrokrlx6/pdfa.ps']
Treating 18 as an optimization candidate
XrefExt(xref=18, ext='.png')
Optimizable images: JPEGs: 0 PNGs: 1
Treating 18 as an optimization candidate
Optimizable images: JBIG2 groups: (0,)
Optimize ratio: 1.00 savings: 0.0%
os.symlink(/tmp/com.github.ocrmypdf.lrokrlx6/optimize.opt.pdf, /tmp/com.github.ocrmypdf.lrokrlx6/optimize.pdf)
/tmp/com.github.ocrmypdf.lrokrlx6/optimize.pdf -> wein111.pdf
Output file is a PDF/A-2B (as expected)
Temporary working files retained at:
/tmp/com.github.ocrmypdf.lrokrlx6
![input](https://user-images.githubusercontent.com/2614431/97867428-20aeb400-1d0e-11eb-8e87-06e34eb678aa.jpg)
[com.github.ocrmypdf.p5dxo11v.tar.gz](https://github.com/jbarlow83/OCRmyPDF/files/5474931/com.github.ocrmypdf.p5dxo11v.tar.gz)

```

Run with verbosity or higher `-v1` to see more detailed logging. This information may be helpful.

**Example file**
[input.jpg](https://user-images.githubusercontent.com/2614431/97867966-235dd900-1d0f-11eb-91f9-af30a9a390a3.jpg)
[com.github.ocrmypdf.p5dxo11v.tar.gz](https://github.com/jbarlow83/OCRmyPDF/files/5474944/com.github.ocrmypdf.p5dxo11v.tar.gz)

**Expected behavior**
The "_ocr.png" is rotated correctly and the output.pdf contains a more or less correct text layer.

**System**
 - OS: Linux
 - OCRmyPDF Version: 11.3.1
 - How did you install ocrmypdf? via arch aur (from pkbuild script:  python setup.py install) 

Thanks a lot. :-)

**Comment by jbarlow83** [comment]
I believe this is fixed in 11.3.2 - or at least, the text layer is correctly aligned for me. Please check and confirm.

**Comment by qupfer** [comment]
Yes, 
works great again with 11.3.2 Thanks
